eth/tracers/api.go

```diff
@@ -809,19 +814,20 @@ func (api *API) standardTraceBlockToFile(ctx context.Context, block *types.Block
 
 			// Swap out the noop logger to the standard tracer
 			writer = bufio.NewWriter(dump)
-			vmConf = vm.Config{
-				Tracer:                  logger.NewJSONLogger(&logConfig, writer),
-				EnablePreimageRecording: true,
-			}
+			*tracer = *logger.NewJSONLogger(&logConfig, writer)
+		} else {
```

Comment:
The `else` condition doesn't need to be added. If the specified `TxHash` using the if condition once and break at the end of the loop, if the `TxHash` is not set using the if condition at each loop.

Reply:
Thanks